### PR TITLE
DEV modify handler/pixel-mask

### PR DESCRIPTION
**Pull request by yugangzhang**
@ericdill  please take a look at the modification of chxtools/handlers

### CODE REVIEW

**Comment by sameera2004** [comment]
@yugangzhang I think that's the same in  https://github.com/NSLS-II-CHX/chxtools/pull/9/files
@ericdill ?

File: chxtools/handlers.py [hunk 1]
```diff
@@ -48,6 +48,18 @@ def __call__(self, seq_id):
...
+        # 1  -- gap
+        # 2  -- dead
+        # 4  -- under-responsive
+        # 8  -- over-responsive
+        # 16 -- noisy
+        pixel_mask = md['pixel_mask']
+        pixel_mask[pixel_mask>0] = 1
+        pixel_mask[pixel_mask==0] = 2
+        pixel_mask[pixel_mask==1] = 0
+        pixel_mask[pixel_mask==2] = 1
```
Comment (ericdill) [inline chxtools/handlers.py:62]: if i understand correctly, this results in a pixel mask where the value of `0` represents the gap between panels, `1` are bad pixels (dead, under-responsive, over-responsive or noisy) and `2` are good pixels?

Comment (sameera2004) [inline chxtools/handlers.py:62]: Yes in line 59 you convert all good pixels to `2`,

**Comment by ericdill** [comment]
Oh, yeah this looks to be teh same code as in #9 . Let's just use that.

**Comment by yugangzhang** [comment]
Thanks sameera and ericdill !

**Pull request closed without merging**